modules/web/js/debugger/tools.js

```diff
@@ -212,16 +213,12 @@ class Tools extends EventChannel {
             if (activeTab && activeTab.getFile()) {
                 const uniqueId = this.getFileUniqueId(activeTab.getFile());
                 const argsString = this.application.browserStorage.get(`launcher-app-configs-${uniqueId}`) || '';
-                const currentArgs = argsString.split(' ');
+                const currentArgs = argsString ? argsString.split(' ') : [];
                 _.each(currentArgs, (arg, i) => {
-                    if (i === 0) {
-                        $('#form-run-application-with-args input[type=\'text\']').get(0).value = arg;
-                    } else {
-                        $(wrapper).append(`<div class="removable">
+                    $(wrapper).append(`<div class="removable">
```

Comment:
seems there is a formatting issue here.

Reply:
No @pahans this is correcly formatted ... this code snippet was inside a else statement not i have remove that else so it is like this.